### ISSUE TITLE
PowerCommands don't work on .NET Core projects

### ISSUE THREAD

**Issue by ildoc** [Microsoft/VS-PPT#61]
"Remove and sort usings on save" and "format document on save" don't work on .NET Core projects.
Are there any workarounds? 
Those are the main reasons why i use PPT...

**Comment by PeterKottas** [comment]
Same for me, what a shame. Hope there will be a way to fix this soon.

**Comment by ildoc** [comment]
I'm using ReSharper now, I don't think I'll use ppt again :(

Il 05 gen 2017 12:19, "Peter Kottas" <notifications@github.com> ha scritto:

**Comment by PeterKottas** [comment]
Yeah, I thought so judging by the date and the fact there was no response. Is anybody actually maintaining this?

**Comment by the-scott-tiger** [comment]
Same here.
These two features should be a core feature of Visual Studio, why we need install there features as a Productivity Power Tools plugin...

**Comment by justcla** [comment]
Thanks for the comments.

ildoc: Thanks for raising the issue. I'll take a look at it.

Scott-the-tiger: Yes, there are lots of features that people believe should be built into the product. And we're always working to include more features. We have a large backlog of great ideas that people would love to see in the product. Please up-vote items on User Voice as it helps us judge the popularity of various features.

PeterKottas: We don't have resources dedicated to maintaining PPT. We have open-sourced about half the extensions and we review and accept pull requests from the community for those ones.
I will see what we can do about open-sourcing more features of PPT so that we can get more community contributions.

**Comment by paulh-rnd** [comment]
Same here. Duplicate of Issue #40?

**Comment by spottedmahn** [comment]
Hello - any updates on this issue?

**Comment by justcla** [comment]
No update. Sorry.
